xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -326,6 +326,31 @@ bool CSystemGUIInfo::GetLabel(std::string& value, const CFileItem *item, int con
 
       return true;
     }
+
+    case SYSTEM_LOCALE_TIMEZONECOUNTRY:
+    {
+        value = CServiceBroker::GetSettingsComponent()->GetSettings()->GetString(CSettings::SETTING_LOCALE_TIMEZONECOUNTRY);
```

Comment:
Indentation looks wrong. Not only here. We use 2 blanks, not 4

Reply:
I applied the Jenkins diff.  That appears to have fixed it.  If not, I can revisit this.